src/olympia/addons/management/commands/process_addons.py

```diff
@@ -41,7 +41,10 @@
         ]},
     'resign_addons_for_cose': {
         'method': sign_addons,
-        'qs': []},
+        'qs': [
+            ~Q(type=amo.ADDON_SEARCH)
```

Comment:
Now that I think about it we probably want to restrict to public add-ons only. No point in re-signing others. The task will filter some of the non-public out because it looks at `current_version`, but that's not enough.